docs/contributing/commit-discipline.md

```diff
@@ -279,6 +279,63 @@ might worry about it breaking.
   you developed this commit or pull request, like "First I tried X" or
   "I changed Y".
 
+#### Don't @-mention people in commit messages
+
+**No GitHub @-mentions:** GitHub makes it tempting to refer to people
+by an @-mention of their GitHub username, like `@gnprice`. Unfortunately
+this is a misfeature in commit messages:
```

Comment:
Suggested-by: Greg Price <gnprice@gmail.com>

Should be `greg@zulip.com`, to match the email address I use in Zulip work.

Reply:
Also, prompted by Tim's comment above, "Co-authored-by" might be the most accurate here, since it's text I originally wrote. Or perhaps "Originally-authored-by"?

Reply:
Got it! I used that email address because it was in a March 2023 commit (26691a3bd1ae6f990295cebf5093653de66ecbef). I'll fix the `Suggested-by` line, too.

And to save you some comment spam, I'll also open a PR on `zulip-mobile` so the docs there benefit from parallel changes here.

Reply:
> I used that email address because it was in a March 2023 commit ([26691a3](https://github.com/zulip/zulip/commit/26691a3bd1ae6f990295cebf5093653de66ecbef)).

Huh indeed. Guess I missed that when I reviewed that PR (#24735).

To see the email address I use on my own commits, you can run `git log --author Greg`, or `git shortlog -es`.